<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestDocumentDialog</class>
 <widget class="QDialog" name="RequestDocumentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblDocumentId">
       <property name="text">
        <string>Порядковый номер</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtDocumentId">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="CStoreComboBox" name="cmbStoreTo"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lblStoreTo">
       <property name="text">
        <string>Получатель</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="edtHistoryNumber"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="lblHistoryNumber">
       <property name="text">
        <string>Номер истории болезни</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblUser">
       <property name="text">
        <string>Заявитель</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="CUserComboBox" name="cmbUser"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblStoreFrom">
       <property name="text">
        <string>Отправитель</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="CStoreComboBox" name="cmbStoreFrom"/>
     </item>
     <item row="1" column="3">
      <widget class="QDateTimeEdit" name="edtDatetime">
       <property name="displayFormat">
        <string>dd.MM.yyyy HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblDate">
       <property name="text">
        <string>Дата создания</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblType">
       <property name="text">
        <string>Тип</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CEnumComboBox" name="cmbType"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblNumber">
       <property name="text">
        <string>Номер</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="edtNumber"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnFinalize">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Утвердить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="CItemListView" name="tblItems"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CEnumComboBox</class>
   <extends>QComboBox</extends>
   <header>library/Enum.h</header>
  </customwidget>
  <customwidget>
   <class>CStoreComboBox</class>
   <extends>QComboBox</extends>
   <header>Pharmacy/ItemListComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CItemListView</class>
   <extends>QTableView</extends>
   <header>library/ItemListView.h</header>
  </customwidget>
  <customwidget>
   <class>CUserComboBox</class>
   <extends>QComboBox</extends>
   <header>Pharmacy/ItemListComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtDocumentId</tabstop>
  <tabstop>edtNumber</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>edtDatetime</tabstop>
  <tabstop>cmbStoreFrom</tabstop>
  <tabstop>cmbStoreTo</tabstop>
  <tabstop>cmbUser</tabstop>
  <tabstop>edtHistoryNumber</tabstop>
  <tabstop>tblItems</tabstop>
  <tabstop>btnFinalize</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RequestDocumentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RequestDocumentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
